canonical-websites/www.ubuntu.com
# 3257 beta nav updates

## Done

- Made bottom section of dropdown white and added keyline above
- Added transparent overlay under dropdown and enable click to close
- Remove links from global nav
- Reformat products tab layout

## QA

- Check out this feature branch
- Run the site using the command `./run serve`
- View the site locally in your web browser at: [http://0.0.0.0:8001/](http://0.0.0.0:8001/)
- Check above issues have been addressed


## Issue / Card

Fixes #3257
